sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Convert from microseconds to milliseconds in CosmosDiagnostics (i.e. replaced "durationInMicroSec" with to "durationInMilliSecs"; and changing type from long to double) - See See [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)
```

Comment:
maybe changing from 
`Convert from microseconds to milliseconds in CosmosDiagnostics (i.e. replaced "durationInMicroSec" with to "durationInMilliSecs"; and changing type from long to double)`

-> 
Convert from `durationInMicroSec` to `durationInMilliSecs ` in durationInMilliSecs CosmosDiagnostics

Reply:
Made all verbs past tense. Good catch!